src/main/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -108,113 +135,170 @@ message WorkerTerminate {
 
+  // type for this event
   Type type = 1;
+
+  // full file path for the file change notification
   string full_path = 2;
+
+  // Name of the function affected
   string name = 3;
 }
 
+// Indicates whether worker reloaded successfully or needs a restart
 message WorkerActionResponse {
+  // indicates whether a restart is needed, or reload succesfully
   enum Action {
     Restart = 0;
     Reload = 1;
   }
   
+  // action for this response
   Action action = 1;
+
+  // text reason for the response
   string reason = 2;
 }
 
+// NOT USED
 message WorkerStatusRequest{
 }
 
+// NOT USED
 message WorkerStatusResponse {
 }
 
+// Host tells the worker to load a Function
 message FunctionLoadRequest {
   // unique function identifier (avoid name collisions, facilitate reload case)
   string function_id = 1;
+
+  // Metadata for the request
   RpcFunctionMetadata metadata = 2;
 }
 
+// Worker tells host result of reload
 message FunctionLoadResponse {
+  // unique function identifier
   string function_id = 1;
+
+  // Result of load operation
   StatusResult result = 2;
   // TODO: return type expected?
 }
 
+// Information on how a Function should be loaded and its bindings
 message RpcFunctionMetadata {
   // TODO: do we want the host's name - the language worker might do a better job of assignment than the host
   string name = 4;
 
+  // base directory for the Function
   string directory = 1;
+  
+  // Script file specified
   string script_file = 2;
+
+  // Entry point specified
   string entry_point = 3;
 
+  // Bindings info
   map<string, BindingInfo> bindings = 6;
-
-  // not adding disabled or excluded as those (currently) are only relevant to host
 }
 
+// Host requests worker to invoke a Function
 message InvocationRequest {
+  // Unique id for each invocation
   string invocation_id = 1;
+
+  // Unique id for each Function
   string function_id = 2;
+
+  // Input bindings (include trigger)
   repeated ParameterBinding input_data = 3;
+
+  // binding metadata from trigger
   map<string, TypedData> trigger_metadata = 4;
 }
 
+// Host requests worker to cancel invocation
 message InvocationCancel {
+  // Unique id for invocation
   string invocation_id = 2;
+
+  // Time period before force shutdown
   google.protobuf.Duration grace_period = 1; // could also use absolute time
 }
 
+// Worker responds with status of Invocation
 message InvocationResponse {
+  // Unique id for invocation
   string invocation_id = 1;
+
+  // Output binding data
   repeated ParameterBinding output_data = 2;
+
+  // data returned from Function (for $return and triggers with return support)
   TypedData return_value = 4;
+
+  // Status of the invocation (success/failure/canceled)
   StatusResult result = 3;
 }
 
+// Used to encapsulate data which could be a variety of types
 message TypedData {
   oneof data {
     string string = 1;
     string json = 2;
     bytes bytes = 3;
     bytes stream = 4;
     RpcHttp http = 5;
-	sint64 int = 6;
+	  sint64 int = 6;
```

Comment:
Space.

Reply:
Will format proto file in https://github.com/Azure/azure-functions-language-worker-protobuf/ repo